pkg/csi/cinder/controllerserver.go

```diff
@@ -43,6 +43,7 @@ import (
 type controllerServer struct {
 	Driver *Driver
 	Clouds map[string]openstack.IOpenStack
+	csi.UnimplementedControllerServer
```

Comment:
why do we need this?

Reply:
it does not compile without, the interface nowadays requires these

Reply:
see for instance aws ebs csi driver commit https://github.com/kubernetes-sigs/aws-ebs-csi-driver/commit/d560c06cf4925d3a6006f4ff065124a2c9041e78#diff-e8a7e777d80a14b455bdbf7aae3f28ad8082ffa0a06579e11cc1af741b5f98f7